tensorflow/python/ops/image_ops_test.py

```diff
@@ -6075,6 +6075,16 @@ def testImageCropAndResize(self):
             crop_size=[1, 1])
         self.evaluate(op)
 
+  def testImageCropAndResizeWithInvalidInput(self):
+    with self.session():
+      with self.assertRaises((errors.InternalError, ValueError)):
+        op = image_ops_impl.crop_and_resize_v2(
+            image=np.ones((1, 1, 1, 1)),
+            boxes=np.ones((11, 4)),
+            box_indices=np.ones((11)),
+            crop_size=[2065374891, 1145309325])
```

Comment:
The issue is that this produces a tensor with a shape that describes a number of elements that can't be represented in an int64_t, do we need such a large size here? Surely we can't execute anything like this right?